tests/Pagination/PaginationPaginatorTest.php

```diff
@@ -174,4 +174,11 @@ public function testSimplePaginatorReturnsRelevantContextInformation()
             'prev_page_url' => '/?page=1', 'from' => 3, 'to' => 4, 'data' => ['item3', 'item4'],
         ], $p->toArray());
     }
+
+    public function testPaginatorGeneratesUrlsWithoutTrailingSlash()
```

Comment:
for example, to check that `http://website.com/test/` will give `http://website.com/test?page=1` too